plugins/inputs/all/all.go

```diff
@@ -39,6 +39,7 @@ import (
 	_ "github.com/influxdata/telegraf/plugins/inputs/diskio"
 	_ "github.com/influxdata/telegraf/plugins/inputs/disque"
 	_ "github.com/influxdata/telegraf/plugins/inputs/dmcache"
+	_ "github.com/influxdata/telegraf/plugins/inputs/dnac"
```

Comment:
hey! thanks for making the updates so far. It looks like tests are failing because the all.go file still has the old name.

```suggestion
	_ "github.com/influxdata/telegraf/plugins/inputs/cisco_dna_center"
```

Reply:
I changed the name there. Tests are running now